framework_tvb/tvb/interfaces/web/controllers/common.py

```diff
@@ -89,6 +89,8 @@
 KEY_SECTION_TITLES = "section_titles"
 KEY_SUBSECTION_TITLES = "sub_section_titles"
 KEY_SIMULATION_NAME = "input_simulation_name_id"
+KEY_PSE_PARAM_1 = "pse_param1"
```

Comment:
We have used before `tvb.core.entities.model.model_burst.RANGE_PARAMETER_1` and `RANGE_PARAMETER_2` Aren't these the same thing?

Reply:
I am still not convinced why we need these?
Can't we use Forms ?

Reply:
Understood today that KEY_PSE_PARAM_1 and KEY_PSE_PARAM_2 should stay in session as separate objects.